README.md

```diff
@@ -1,302 +1,293 @@
-# Origami Tracking component
+# o-tracking
 
-Include in your product to send tracking requests to the Spoor API.
+Include in your product to send tracking requests to the [Spoor API](https://spoor-docs.herokuapp.com/).
 
-- [The spoor ecosystem](#the-spoor-ecosystem)
 - [Usage](#usage)
-- [Events](#events)
-- [Parameters](#parameters)
+- [How To View The Data](#how-to-view-the-data)
```

Comment:
Looks like this is missing some headings, are we linting it?

Reply:
Fixed in https://github.com/Financial-Times/o-tracking/pull/309